modules/tinymce/src/themes/silver/test/ts/browser/window/SilverInlineDialogPositionTest.ts

```diff
@@ -119,6 +119,45 @@ describe('browser.tinymce.themes.silver.window.SilverInlineDialogPositionTest',
     });
   });
 
+  context('TINY-9554: dialog position with editor in fixed container', () => {
+    const setupElement = () => {
+      const container = SugarElement.fromHtml('<div style="position: fixed; top: 150; left: 150;"></div>');
+      const element = SugarElement.fromTag('textarea');
+
+      Insert.append(SugarBody.body(), container);
+      Insert.append(container, element);
+
+      return {
+        element,
+        teardown: () => {
+          Remove.remove(element);
+          Remove.remove(container);
+        }
+      };
+    };
+
+    const hook = TinyHooks.bddSetupFromElement<Editor>({
+      base_url: '/project/tinymce/js/tinymce',
+      height: 400,
+      width: 650,
+      menubar: false
+    }, setupElement, []);
+
+    it('TINY-9554: test position', async () => {
```

Comment:
Maybe elaborate this label a bit it's testing what exactly?

Reply:
done